composer.json

```diff
@@ -22,6 +22,7 @@
         "symfony/form": "^2.3.5 || ^3.0",
         "symfony/http-foundation": "^2.3 || ^3.0",
         "symfony/property-access": "^2.3 || ^3.0",
+        "symfony/security": "^2.3 || ^3.0",
```

Comment:
@greg0ire If it is only used by tests, shouldn't it be on require-dev? Same happens for the twig-bridge